ttnn/ttnn/__init__.py

```diff
@@ -30,6 +30,10 @@ def get_bool_env_var(name, default):
 if TTNN_ENABLE_FAST_RUNTIME_MODE:
     logger.info(f"ttnn: fast runtime mode was enabled")
 
+TTNN_ENABLE_LOGGING = get_bool_env_var("TTNN_ENABLE_LOGGING", "False")
+if TTNN_ENABLE_LOGGING:
+    logger.info(f"ttnn: enabled logging (and disabled fast runtime mode)")
```

Comment:
Does "fast runtime mode" here basically means "will wait for all devices to finish all operations after each line of a script?

Reply:
No, it disables features that aren't necessary like validation and graph tracing